src/logging.c

```diff
@@ -73,9 +93,15 @@ void rcutils_log(
 }
 
 void rcutils_logging_console_output_handler(
-  rcutils_log_location_t * location,
+  rcutils_log_location_t * location_,
   int severity, const char * name, const char * format, va_list * args)
 {
+  rcutils_log_location_t * location = location_;
```

Comment:
Why are two separate variables needed here? Just having an argument name `location` and optionally pointing it to the dummy_location should be fine.

Reply:
I thought there'd be a chance the user was counting on it to stay null and didn't want to change it under their nose, but I suppose I could have changed it back afterwards. I avoided modifying it in 3049cb0afec856136e0348d7ef04614c0ed915c1 (many other things have changed in the meantime though)